native/src/lib.rs

```diff
@@ -123,9 +123,9 @@ fn bls_public_key_to_bbs_key(mut cx: FunctionContext) -> JsResult<JsArrayBuffer>
 /// `signature_context`: `Object` the context for the signature creation
 /// The context object model is as follows:
 /// {
-///     "secretKey": ArrayBuffer           // The private key of signer
-///     "publicKey": ArrayBuffer           // The public key of signer
-///     "messages": [String, String],      // The messages to be signed as strings. They will be hashed with SHAKE-256
+///     "secretKey": ArrayBuffer                // The private key of signer
+///     "publicKey": ArrayBuffer                // The public key of signer
+///     "messages": [ArrayBuffer, ArrayBuffer], // The messages to be signed as strings. They will be hashed with Blake2b
```

Comment:
Did this change change it to Blake2b hash or is this just a minor update for accuracy? From what I can tell the hash algorithm used in these changes hasn't changed.

(Thinking it might also be the underlying Ursa implementation changed)

Reply:
Na we are just shifting the responsibility for the caller to encode the string to the bytes

Reply:
I'm confused what the `SHAKE-256` => `Blake2b` change in the comment is for then. Do you have additional details on why this has changed?

Reply:
Similar to how most other cryptographic libraries do this

Reply:
Ah right good question the comment was out of date we have been using blake2b since the first release